test/asyncMotionProfileControllerTests.cpp

```diff
@@ -66,12 +64,349 @@ class AsyncMotionProfileControllerTest : public ::testing::Test {
   SkidSteerModel *model;
   MockAsyncMotionProfileController *controller;
 
-  FILE *leftPathFile;
-  FILE *rightPathFile;
-  char *leftFileBuf;
-  char *rightFileBuf;
-  size_t leftFileSize;
-  size_t rightFileSize;
+  std::stringstream squigglesPathFile;
+  std::stringstream leftPathFile;
+  std::stringstream rightPathFile;
+  std::string squigglesFileBuf;
+  const std::string left = "dt,x,y,position,velocity,acceleration,jerk,heading\n"
```

Comment:
That makes sense, they're definitely annoying to keep around in the test source. Think it would be best to give them their own cpp file and create a simple header file to extern them for this test?

Reply:
I would prefer if they aren't in a code-type file at all. Just `.txt` files would work fine. You can load from the file during the setup phase of the test suite.